katsdptelstate/test/test_rdb_handling.py

```diff
@@ -61,15 +56,14 @@ def test_writer_reader(self):
         self.assertEqual(self.rdb_writer.save(self.base('broken.rdb'), keys=['does_not_exist'])[0], 0)
 
         local_tr = TabloidRedis()
-        load_from_file(local_tr, self.base('all.rdb'))
```

Comment:
I have a question mark around this duplicate line. Is that still required to pass? Using the callback seems to make it fail, so I removed it.

Reply:
I think it was accidentally left behind when I added the line below it to validate the return value. Fine to delete it.